tests/test_scheduler.py

```diff
@@ -59,97 +66,154 @@ def test_simple_interval_scheduler(start_scheduler, start_cli):
 
 
 @pytest.mark.skipif(os.getenv("CI") == "true", reason="test skipped on CI")
-def test_multiple_schedulers(start_scheduler, start_cli):
-    from tests.scheduler_configs.simple_1 import broker, write_loaded_at
+def test_multiple_schedulers(stub_broker, stub_worker, schedulers):
```

Comment:
maybe you don't need to declare `schedulers` as a fixture as it's used only by this test.